distribution/src/conf/file.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+stockQuoteEP=http://localhost:9000/services/SimpleStockQuoteService
```

Comment:
Shall we add a new line at the EOF

Reply:
Fix with the commit https://github.com/wso2/micro-integrator/commit/ccb931501ef0434ae0955d1b64e04d8c5845360b